ksqldb-api-client/src/main/java/io/confluent/ksql/api/client/impl/ClientOptionsImpl.java

```diff
@@ -265,7 +275,13 @@ public ClientOptions copy() {
         keyStorePath, keyStorePassword, keyPassword, keyAlias,
         basicAuthUsername, basicAuthPassword,
         executeQueryMaxResultRows, http2MultiplexingLimit,
-        requestHeaders);
+        requestHeaders,
+        consistencyLevel);
+  }
+
+  @Override
+  public ConsistencyLevel getConsistencyLevel() {
+    return consistencyLevel;
```

Comment:
What's the default consistency level? In the client implementation, the default behavior is eventual, but according to this class, the default level is `null`; is that intended?

Reply:
Fixed